mlir_graphblas/tests/test_mlir_builder.py

```diff
@@ -10,8 +10,12 @@
 from mlir_graphblas.sparse_utils import MLIRSparseTensor
```

Comment:
Another thing that's not part of this PR: `import mlir` isn't used in this file, so we should probably get rid of that as well.